main/src/main/java/org/mobilitydata/gtfsvalidator/validator/AgencyConsistencyValidator.java

```diff
@@ -78,17 +78,18 @@ public void validate(NoticeContainer noticeContainer) {
                   GtfsAgency.AGENCY_ID_FIELD_NAME));
         }
       }
-    }
-    // agency_timezone field is required and it must be the same for all agencies.
-    ZoneId commonTimezone = agencyTable.getEntities().get(0).agencyTimezone();
-    for (int i = 1; i < agencyCount; ++i) {
-      GtfsAgency agency = agencyTable.getEntities().get(i);
-      if (!commonTimezone.equals(agency.agencyTimezone())) {
-        noticeContainer.addValidationNotice(
-            new InconsistentAgencyTimezoneNotice(
-                agency.csvRowNumber(), commonTimezone.getId(), agency.agencyTimezone().getId()));
+      // agency_timezone field is required and it must be the same for all agencies.
+      ZoneId commonTimezone = agencyTable.getEntities().get(0).agencyTimezone();
+      for (int i = 1; i < agencyCount; ++i) {
+        GtfsAgency agency = agencyTable.getEntities().get(i);
+        if (!commonTimezone.equals(agency.agencyTimezone())) {
+          noticeContainer.addValidationNotice(
+              new InconsistentAgencyTimezoneNotice(
+                  agency.csvRowNumber(), commonTimezone.getId(), agency.agencyTimezone().getId()));
+        }
       }
     }
+
```

Comment:
I don't see any difference here? is it just white space?

Reply:
It looks like a new line.